src/coreComponents/constitutive/fluid/PVTFunctions/CO2SolubilityFunction.cpp

```diff
@@ -42,56 +42,56 @@ constexpr real64 acoef[] =
  8.93353441e-5, 7.88998563e-5, -1.66727022e-2, 1.398, 2.96e-2};
 
 
-real64 ff( real64 const & T, real64 const & P, real64 const & V_r )
```

Comment:
`T` is standard short for temperature?
`P` is standard short for pressure.
`V_r` is probably `volume_r`...don't know what r stands for.
@yue-2018 @francoishamon @klevzoff ?

at any rate, don't think this should be applied here.